Dockerfile

```diff
@@ -0,0 +1,46 @@
+#
```

Comment:
move it under `presto-docker`

Reply:
sorry, nvm.

Reply:
Actually while this seemingly makes sense, it doesn't really work in practice. We need the build context to include the `target` directories of `presto-server`, `presto-cli`, and `presto-docker` and while we can move the `Dockerfile` to `presto-docker` the file then doesn't make much sense as the directories in it then don't line up. Additionally we still need the `.dockerignore` file in the root as it's referenced only from the build context.